packages/core/src/App/Containers/RealAccountSignup/add-or-manage-accounts.jsx

```diff
@@ -58,13 +58,12 @@ const AddOrManageAccounts = props => {
         setActiveIndex(index);
     };
 
-    const manageOrChangeAccount = (obj, setSubmitting) => {
+    const manageOrChangeAccount = obj => {
         setLoading(true);
         Object.entries(obj).map(([key, value]) => {
             if (key === 'fiat') {
                 setCurrency(value)
                     .then(response => {
-                        setSubmitting(false);
```

Comment:
no need to set `isSubmitting` to false here and at line 83 because by the time we set it here, `ChangeAccountCurrency, AddCurrency, AddCryptoCurrency` are unmounted already.
So this change helps us fix the following console warning visible in dev environment, for example:
```
Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    in Formik (created by ChangeAccountCurrency)
    in ChangeAccountCurrency (created by AddOrManageAccounts)....
```